screens/HomeScreen.js

```diff
@@ -23,13 +25,39 @@ export default function HomeScreen() {
 
   let totalExpense = _sumBy(ALL_EXPENSES, 'amount');
 
+  const handleSubmitForm = () => {
+    //check whether the name is not empty and the amount is not negative
+    if (name !== '' && amount > 0) {
+      // single expense object
+      const expense = { name, amount };
+      // do not override previous values in the array
+      // use spread operator to access previous values
+      setExpenses([...expenses, expense]); // !!! INVALID ATTEMPT TO SPREAD NON-ITERABLE INSTANCE.
```

Comment:
Is the issue #5 connected to this?

Reply:
it will next time, I can't find a way to edit the commit at this point .

Reply:
Hm do not understand. Ok than show the problem in #5 more clear?